modules/api/src/main/java/org/eclipse/xpanse/api/config/cache/CaffeineCacheConfig.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.api.config.cache;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import java.util.concurrent.TimeUnit;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cache.caffeine.CaffeineCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * The configuration class of Caffeine Cache.
+ */
+@Configuration
+public class CaffeineCacheConfig {
```

Comment:
since this cache manager is just for AZ cache. We need a specific naming. 
Also it is better we move this class to "cache" maven module.